wordpress-mobile/WordPress-iOS
# Gutenberg related changes to land after 16.2 App cut

This PR is for merging Gutenberg related changes to WPiOS `develop` that were ongoing during the Gutenberg Editor 1.41.0 code freeze

PLEASE DO NOT MERGE until after 16.2 Release is cut on WPiOS `develop`. Thank you!

To test:
See test steps: https://github.com/wordpress-mobile/WordPress-iOS/pull/15122

PR submission checklist:
